src/classes/CON_ContactMerge_CTRL.cls

```diff
@@ -726,9 +726,7 @@ public with sharing class CON_ContactMerge_CTRL {
             
             System.SavePoint sp = Database.setSavepoint();
             try {
-                Database.DMLOptions dml = new Database.DMLOptions();
-                dml.DuplicateRuleHeader.AllowSave = true;
-                Database.SaveResult sr = Database.update(master, dml);
+                Database.SaveResult sr = UTIL_DMLService.updateRecord(master);
```

Comment:
@mpusto @joy-sfdo here is a scenario (contact merge) where we were previously forcing AllowSave to true.  Are we sure we want to now change this to be based off the setting?  I'm concerned that contact merges will now fail that used to succeed, since we defaulting our new setting Respect_Duplicate_Rule_Settings__c to false.

Reply:
@davidhabib -- Chatted with @mpusto, we'll consider this a special case so I'll set the AllowSave to true.